### DISCUSSION TITLE
[Bug] crowbar won't swing

### DISCUSSION THREAD

**Discussion by GhostsModels** [EtherealEmil/UsualScrap#37] [General]
I think this has to do with me dying with the crowbar after blowing myself up by hitting the explosive barrel

**Comment by GhostsModels** [comment]
this is only for me, not other players. the same item works for them

**Comment by EtherealEmil** [comment]
I'll try to replicate it when I get home but if I can't then I cannot really help. Previously when playing, I've never seen that bug occur and I haven't touched the crowbar code since.

**Comment by EtherealEmil** [comment]
I haven't had any luck replicating this bug and haven't gotten any errors hitting a tank with a crowbar inside or outside the ship. This could be a mod compat issue maybe, or a very situational bug, or just a small lethal company hiccup, either way I can't fix what I can't find.
